tools/sdk-tools/adpcm/util.c

```diff
@@ -60,13 +60,13 @@ char *ReadPString(FILE *ifile)
     u8 c;
     char *st;
 
-    fread(&c, 1, 1, ifile);
+    if (fread(&c, 1, 1, ifile)){};
```

Comment:
What. Is there an "unused result" warning here? Use `(void) fread(...);` instead of this ebin construction. Same for the ones below.

Reply:
so you'd prefer the (void) approach here. Should be easy enough to tweak.

Reply:
Using (void) here for whatever reason doesn't actually silence the warning. The way this does it does, but it'll need to actually check the return value here and fail if incorrect.

Reply:
Resolved by turning these into error-checkers.